flake.nix

```diff
@@ -68,7 +68,7 @@
               export KBUILD_DIR=".kbuild"
               patchShebangs .
               make build-backends VERSION="${k_release}" POETRY_RUN=
-              kmir init $(kbuild which llvm)
+              kmir gen-glr-parser $(kbuild which llvm) --verbose
```

Comment:
With Pyk PR#785 https://github.com/runtimeverification/pyk/pull/785 Merged, this line should be removed. The parser should be built at the `kbuild kompile llvm` step.